### ISSUE TITLE
Showing incorrect time

### ISSUE THREAD

**Issue by freznice** [rafcontreras/Ubersicht-World-Clock-Widget#10]
I really like this clock face. But there is one fundamental problem, after few minutes for example after refreshing the widget, the displayed time gets off from the real time. Initially it's off by 30 minutes but later on the difference increases to hours.
Anyone seen this problem?

**Comment by SteveMattan** [comment]
Yes. I stopped using it because of this. For a while it was just the day that was off, and I didn't mind that all that much. But once the time was no longer accurate it was rendered useless.
